prusti-viper/src/encoder/errors/error_manager.rs

```diff
@@ -698,6 +715,35 @@ impl<'tcx> ErrorManager<'tcx> {
                     error_span
                 )
             }
+            ("exhale.failed:insufficient.permission" | "assert.failed:insufficient.permission", ErrorCtxt::NotEnoughTimeCredits) => {
+                PrustiError::verification("Not enough time credits.".to_string(), error_span)
+            }
+            ("exhale.failed:insufficient.permission" | "assert.failed:insufficient.permission", ErrorCtxt::NotEnoughTimeReceipts) => {
+                PrustiError::verification("Not enough time receipts.".to_string(), error_span)
+            }
+
+            // TODO: this is a temporary fix for detecting time credits/receipts errors
```

Comment:
Please add a more detailed description to the TODO. We might know (roughly) what the problem is now, but we might forget :)